pkg/build/android.go

```diff
@@ -122,6 +123,8 @@ func (a android) build(params Params) (ImageDetails, error) {
 		return details, fmt.Errorf("failed to generate signature: %w", err)
 	}
 
+	details.BuildSrcPath = buildCfg.BuildSrcPath
```

Comment:
`buildCfg.BuildSrcPath` will be relative, right? It cannot be absolute anyway though because it's `syz-ci` that will determine the kernel checkout location and pass it here. Actually, does the relative part vary or it is always the same for builds in `android.go`?

Also note that the returned `details.BuildSrcPath` must be **absolute** (because of `mgrcfg.KernelBuildSrc`), so you'd better do `filepath.Join` with `params.KernelDir` or whatever is better suitable here.

Reply:
Yes, `buildCfg.BuildSrcPath` will be relative. It can vary across kernel versions/devices. It's usually something like `out/mixed/android13-5.10/aosp` for 13/5.10, but will change for different Android/kernel versions.

Ack on the need to be absolute (and thus join with `params.KernelDir`.